src/components/posts/PostDetail.css

```diff
@@ -22,6 +22,6 @@
   font-size: 1.5rem;
 }
 
-.btn-primary {
+.postDetail.btn-primary {
```

Comment:
Changing the class name to `.postDetail.btn-primary` will select anything that has BOTH the classes `postDetail` and `btn-primary`. If you put a space between the two selectors though, so that it reads like `.postDetail .btn-primary`, then that will select any `btn-primary` that is nested at any level within a `postDetail`, so this is probably the selector we want here.